graphics/opencv4/Portfile

```diff
@@ -1,46 +1,49 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           cmake 1.1
 PortGroup           compiler_blacklist_versions 1.0
 PortGroup           github 1.0
 
-github.setup        opencv opencv 4.5.0
+github.setup        opencv opencv 4.5.1
 name                opencv4
-revision            1
+revision            0
 categories          graphics science
 platforms           darwin
 license             BSD
 
-maintainers         \
-                    {stromnov @stromnov} \
-                    {@mascguy} \
-                    openmaintainer
+maintainers         {stromnov @stromnov} openmaintainer
```

Comment:
You have removed the maintainer @mascguy. Please restore him.

Reply:
Thanks, done.